app/presenters/collection_presenter.rb

```diff
@@ -0,0 +1,28 @@
+class CollectionPresenter
```

Comment:
Another thing I am concerned about is that the name for a base item presenter class called BasePresenter, whilst class for collection is CollectionPresenter. Feels a bit confusing. What if they were ItemPresenter and CollectionPresenter, or BasePresenter and BaseCollectionPresenter. What do you think? @timurvafin @vast